components/dash-html-components/scripts/generate-components.js

```diff
@@ -263,14 +269,19 @@ const ${Component} = (props) => {
     if(props.loading_state && props.loading_state.is_loading) {
         dataAttributes['data-dash-is-loading'] = true;
     }
+    /* remove unnecessary onClick event listeners  */
+    let isStatic = typeof props.id === "undefined" ? true : false
```

Comment:
There is no need for the `? true : false` as it's the condition itself.
Simplified:
```javascript
const isStatic = props.disable_n_clicks || typeof props.id === "undefined";
```

Reply:
oh yes, that's better.  Thanks!